Maple2.Tools/Collision/Polygon.cs

```diff
@@ -9,7 +9,7 @@ namespace Maple2.Tools.Collision;
 public abstract class Polygon : IPolygon {
     public abstract Vector2[] Points { get; }
 
-    public virtual bool Contains(in Vector2 point) {
+    public virtual bool Contains(in Vector2 point, float epsilon = 1e-5f) {
```

Comment:
Unused `epsilon`, likely the correct place is changing the cases to `case > epsilon:` and `case < -epsilon:`, but would need to test further.